public_suffix_list.dat

```diff
@@ -12344,6 +12344,11 @@ homeoffice.gov.uk
 // Submitted by Zoltan Egresi <egresi@globehosting.com>
 ro.im
 
+// Gnomio LLC
+// Submitted by Gnomio Tech Team <admin@gnomio.com>
+gnomio.com
```

Comment:
This looks dangerous. If you declare gnomio.com a pubic suffix you won't be able to set cookies on gnomio.com anymore. Are you sure that's what you want?